Build fails if Cargo.lock is not used

When installing via a local clone, I get a warning/compile error.
```
$ cargo install --path . -j8 --force
  Installing tealdeer v1.4.1 (~/tools/tealdeer)
    Updating crates.io index
   Compiling tealdeer v1.4.1 (~/tools/tealdeer)
warning: use of deprecated associated function `clap::Arg::<'help>::hidden`: Replaced with `Arg::hide`
   --> src/main.rs:125:9
    |
125 |         hidden = true
    |         ^^^^^^
    |
    = note: `#[warn(deprecated)]` on by default

error[E0599]: no variant or associated item named `HelpRequired` found for enum `AppSettings` in the current scope
  --> src/main.rs:59:31
   |
59 | #[clap(setting = AppSettings::HelpRequired)]
   |                               ^^^^^^^^^^^^ variant or associated item not found in `AppSettings`

For more information about this error, try `rustc --explain E0599`.
warning: `tealdeer` (bin "tldr") generated 1 warning
error: failed to compile `tealdeer v1.4.1 (~/tools/tealdeer)`, intermediate artifacts can be found at `~/tools/tealdeer/target`

Caused by:
  could not compile `tealdeer` due to previous error; 1 warning emitted
```

tealdeer hash is 1a3624d0116, cargo version:
```
$ cargo --version
cargo 1.57.0 (b2e52d7ca 2021-10-21)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,7 +22,7 @@ path = "src/main.rs"
 ansi_term = "0.12.0"
 app_dirs = { version = "2", package = "app_dirs2" }
 atty = "0.2"
-clap = { version = "3.0.0-beta.5", features = ["std", "derive", "suggestions" ], default-features = false }
+clap = { version = "3.0.0-rc.4", features = ["std", "derive", "suggestions" ], default-features = false }
 env_logger = { version = "0.9", optional = true }
 log = "0.4"
 reqwest = { version = "0.11.3", features = ["blocking", "rustls-tls", "rustls-tls-native-roots"], default-features = false }

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,7 +22,7 @@ path = "src/main.rs"
 ansi_term = "0.12.0"
 app_dirs = { version = "2", package = "app_dirs2" }
 atty = "0.2"
-clap = { version = "3.0.0-rc.4", features = ["std", "derive", "suggestions" ], default-features = false }
+clap = { version = "3.0.0-rc.4", features = ["std", "derive", "suggestions", "color"], default-features = false }
 env_logger = { version = "0.9", optional = true }
 log = "0.4"
 reqwest = { version = "0.11.3", features = ["blocking", "rustls-tls", "rustls-tls-native-roots"], default-features = false }

--- a/docs/src/usage.txt
+++ b/docs/src/usage.txt
@@ -15,7 +15,6 @@ OPTIONS:
     -f, --render <FILE>          Render a specific markdown file
     -p, --platform <PLATFORM>    Override the operating system [possible values: linux, macos,
                                  windows, sunos, osx]
-    -o, --os <OS>                Deprecated alias of `platform`
     -L, --language <LANGUAGE>    Override the language
     -u, --update                 Update the local cache
     -c, --clear-cache            Clear the local cache

--- a/src/main.rs
+++ b/src/main.rs
@@ -57,6 +57,7 @@ const ARCHIVE_URL: &str = "https://tldr.sh/assets/tldr.zip";
 #[clap(about = "A fast TLDR client", author, version)]
 #[clap(setting = AppSettings::ArgRequiredElseHelp)]
 #[clap(setting = AppSettings::DeriveDisplayOrder)]
+#[clap(setting = AppSettings::DisableColoredHelp)]
 #[clap(
     after_help = "To view the user documentation, please visit https://dbrgn.github.io/tealdeer/."
 )]

--- a/src/main.rs
+++ b/src/main.rs
@@ -56,7 +56,6 @@ const ARCHIVE_URL: &str = "https://tldr.sh/assets/tldr.zip";
 #[derive(Parser, Debug)]
 #[clap(about = "A fast TLDR client", author, version)]
 #[clap(setting = AppSettings::ArgRequiredElseHelp)]
-#[clap(setting = AppSettings::HelpRequired)]
 #[clap(setting = AppSettings::DeriveDisplayOrder)]
 #[clap(
     after_help = "To view the user documentation, please visit https://dbrgn.github.io/tealdeer/."
@@ -122,7 +121,7 @@ struct Args {
         long = "markdown",
         short = 'm',
         requires = "command_or_file",
-        hidden = true
+        hide = true
     )]
     markdown: bool,
 

--- a/src/main.rs
+++ b/src/main.rs
@@ -92,7 +92,7 @@ struct Args {
         short = 'o',
         long = "os",
         possible_values = ["linux", "macos", "windows", "sunos", "osx"],
-        hide_possible_values = true,
+        hide = true
     )]
     os: Option<PlatformType>,
 

